### PR TITLE
Resolve-McasInfrequentCountryAlerts playbook added

### PR DESCRIPTION
**Pull request by Sebmolendijk**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+|Check_group_membership|Microsoft Graph|/users/{user UPN}/checkMemberGroups|Directory.Read.All|
+|Get_user_AAD_risk_status|Microsoft Graph|/riskyUsers/{user AAD object Id}|IdentityRiskyUser.Read.All|
+|Resolve_Cloud_App_Security_alert|MCAS API|/cas/api/v1/alerts/resolve/||
+
+<br>
+
+### Additional resources
+
+* Complete explanation and demonstration of this playbook in [this video](https://youtu.be/ql8x4rC6m9A).
+* [Registering a service principal in Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-azure-ad-and-create-a-service-principal)
```
Comment (sarah-yo) [inline Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md:30]: Take out the "en-us" from the URL

File: Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* [Microsoft Graph permissions reference](https://docs.microsoft.com/en-us/graph/permissions-reference)
```
Comment (sarah-yo) [inline Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md:31]: Ditto

File: Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* [Create an MCAS API token](https://docs.microsoft.com/en-us/cloud-app-security/api-tokens)
```
Comment (sarah-yo) [inline Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md:32]: Ditto

File: Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md (hunk 1 above)
Comment (Sebmolendijk) [reply Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md]: Done

File: Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md (hunk 2 above)
Comment (Sebmolendijk) [reply Playbooks/Resolve-McasInfrequentCountryAlerts/readme.md]: Done

**Pull request merged**